content/en/event-driven_architecture.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Event-Driven Architecture
+status: Feedback Appreciated
+category: concept
+---
+## What it is
+Event-driven architecture is a software architecture that promotes the creation, processing, and consumption of events. An event can be seen as any change of state or other significant occurrence in hardware or software. This architecture creates the structure in which events can be properly routed from their source to the desired receivers.
+## Problem it addresses
+As more data is becomes real-time, finding reliable ways to ensure that events are captured amd routed to the service that must process event request gets increasingly challenging. Many applications need a way to guarantee event delivery to ensure that the application functions designed. With multiple event sources and event receiver, it also becomes challenging to ensure that events are properly routed to their intended recipient services.
+## How it helps
+Event-driven architecture establishes the event producers (source) consumers (receiver). This architecture ensures that services remain decoupled and that events are properly routed from the producer to the consumer. The producer will take the incoming event, usually by HTTP protocol and then route the event information.
```

Comment:
Same here. This is marked as resolved but the typos are still here. It should be either:

- An event-driven architecture establishes OR
- Event-driven architectures establish 

An "and" appears to be missing and so is a comma. If you disagree, could you please explain why? Otherwise, we'll be running in circles. I'll keep correcting the same things, and you'll keep resolving without committing :)